src/lib/support/Span.h

```diff
@@ -173,6 +173,28 @@ inline bool Span<T>::data_equal(const FixedSpan<U, N> & other) const
     return (size() == other.size()) && (empty() || (memcmp(data(), other.data(), size() * sizeof(T)) == 0));
 }
 
+/**
+ * @brief Returns true if the `span` could be used to access some data,
+ *        false otherwise.
+ * @param[in] span The Span to validate.
+ */
+template <typename T>
+inline bool is_span_usable(const Span<T> & span)
```

Comment:
Dominant function naming style is `IsSpanUsable()`  or `Span::IsUsable()`.

Reply:
Renamed `IsSpanUsable()`